pkg/cfn/builder/iam_helper.go

```diff
@@ -146,6 +146,10 @@ func createRole(cfnTemplate cfnTemplate, clusterIAMConfig *api.ClusterIAM, iamCo
 		cfnTemplate.attachAllowPolicy("PolicyXRay", refIR, xRayStatements())
 	}
 
+	if api.IsEnabled(iamConfig.WithAddonPolicies.CloudWatch) {
+		cfnTemplate.attachAllowPolicy("PolicyCloudwatch", refIR, cloudwatchStatements())
```

Comment:
Can you add a test for this, please?

Reply:
sorry I dont know how - I just tested manually